TwitchPlaysAssembly/Src/TwitchGame.cs

```diff
@@ -588,6 +594,15 @@ private IEnumerator SendDelayedMessage(float delay, string message, Action callb
 		callback?.Invoke();
 	}
 
+	private IEnumerator AdjustFindClaimLimit()
+	{
+		while(true)
+		{
+			yield return new WaitForSeconds(300f);
```

Comment:
The amount of time should probably also be a setting.

Reply:
I added this, as well as if the number was less than 1, it would not add any to `FindClaimUse`